CHANGES

```diff
@@ -10,6 +10,13 @@
 
 There will be a future change to where the frontend bundle will be loaded asynchronously. This will be a breaking change that can affect custom plugins that access certain globals in the django template. Please see https://forum.sentry.io/t/breaking-frontend-changes-for-custom-plugins/14184 for more information.
 
+21.7.1
+------
+
+- If you are upgrading from a release prior to 10.0.0 you need to first upgrade
+  to 21.6.1 and run migrations. In 21.7.1 the ability to upgrade from south based
+  migrations was removed.
```

Comment:
This part needs to be moved to top, before `21.6.1` heading. I'd also add a `### Breaking Change` sub-heading like the one in `21.6.0` and put a link to https://develop.sentry.dev/self-hosted/#hard-stops